VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -34,7 +35,7 @@ public CoursesAsyncLoader(Context context, Bundle args){
     public AsyncTaskResult<List<EnrolledCoursesResponse>> loadInBackground() {
 
         Api api = new Api(getContext());
-        if (this.oauthToken == null){
+        if (Config.getInstance().getSocialSharingConfig().isEnabled() && this.oauthToken == null) {
```

Comment:
@aleffert  This class itself mostly takes care of social sharing feature. I believe the developer null-checked oauthToken which is specifically Facebook accessToken.  Facebook token on My Courses will come into picture only if social sharing is enabled, so I added that condition. 

Not sure, if removing the condition itself will have any side-effects?  Later, once we start testing social sharing feature, we will take care of it.